### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by emperor9** [Azure/Multivariate-AD#1]
I did not see the code and data set in this, can you tell me where I can find it?

**Comment by TBN1zanzan** [comment]
> I did not see the code and data set in this, can you tell me where I can find it?

Do you know how to find it?

**Comment by MrZhangKY** [comment]
yes, maybe this is just a github practice project for the author~

**Comment by zjy5755202** [comment]
fake project without code and dataset

**Comment by zyz760543032** [comment]
fake code and dataset

**Comment by wQvaale** [comment]
If you're looking for an implementation of the [Multivariate Time-series Anomaly Detection via Graph Attention Network](https://arxiv.org/abs/2009.02040), a fellow student (@axeloh) and I wrote an implementation in PyTorch a couple years back for our master's thesis. It's not actively maintained, but please do check it out!

https://github.com/ML4ITS/mtad-gat-pytorch
